modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+Each user of a project is assigned to a cluster role based on their access control. These cluster roles are mentioned in the *Project* view > *Project access* tab > *Role* drop down. By default these roles are *Admin*, *Edit*, and *View*. To add or edit the cluster roles for a project, you can customize the YAML code of the cluster.
```

Comment:
See grammatical changes: 

The users of a project are assigned to a cluster role based on their access control. These cluster roles are mentioned in the *Project* view > *Project access* tab > *Role* drop down. By default, these roles are *Admin*, *Edit*, and *View*. To add or edit the cluster roles for a project, you can customize the YAML code of the cluster.